### PR TITLE
Add gitignore for python generated files

### PR DESCRIPTION
**Pull request by zhoonit**
**Changes proposed in this PR:**
- Add python generated extensions to gitignore

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [x]Skipped
2. Run test: [ ]Passed [ ]Failed [x]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,8 @@
 *.so
 *.o
 *.a
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
```
Comment (kparichay) [inline .gitignore:28]: **/__pycache__

Comment (zhoonit) [reply .gitignore:28]: Do you have any reason for the suggestion?
The patch is from github reference.

Comment (kparichay) [reply .gitignore:28]: yes, `__pycache__` ignores pycache at the root folder only.
`**/__pycache__` ignores pycache folder in the whole repository.

**Review by kparichay — APPROVED** [review]
LGTM

**Comment by kparichay** [comment]
@zhoonit please rebase and merge.

**Pull request merged**